### DISCUSSION TITLE
copilot doesn't work when VS2022 is running its debugger

### DISCUSSION THREAD

**Discussion by mattetti** [community/community#41037] [Copilot Conversations]
### Copilot Visual Studio - debugger

Bug

### Body

GitHub Copilot   1.62.0.0 (v1.62.0.0@00eaa22e0)
GitHub Copilot Agent   1.62.7527 (v1.62.0)
Microsoft Visual Studio Community 2022 (64-bit) - Version 17.3.4
Visual C++ 2022  
Microsoft Visual C++ 2022

It's quite common to run a C++ program in debug mode, make some changes and hot reload it. A good example of such a flow would be when using Unreal Engine. 

## Issue
Unfortunately, copilot isn't making any inline suggestions (automatically or when manually triggered) and when trying to open the GH panel, the logo keeps spinning saying "loading solutions".

## Expectation
I'd expect copilot to just work, just like autocompletion. 

## Feedback

Maybe the debugger is preventing the extension from working properly, in which case, I'd suggest to at least give the user some feedback that copilot is disabled when the debugger is on. Ideally, this is just an unexpected bug and we can have this feature working while the debugger is on.

**Comment by ossanna16** [comment]
Hi there @mattetti and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
